### ISSUE TITLE
Uncaught TypeError: atom.workspace.getActivePaneItem(...).getFileName is not a function

### ISSUE THREAD

**Issue by djit** [icetee/remote-ftp#853]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.5
**Thrown From**: [Remote-FTP](https://github.com/mgrenier/remote-ftp) package 1.0.0

### Stack Trace

Uncaught TypeError: atom.workspace.getActivePaneItem(...).getFileName is not a function

```
At /Applications/Atom.app/Contents/Resources/app/src/pane.js:1095

TypeError: atom.workspace.getActivePaneItem(...).getFileName is not a function
    at Main.fileSaved (/packages/Remote-FTP/lib/remote-ftp.js:111:62)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at TextBuffer.module.exports.TextBuffer.saveAs (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:1142:33)
    at TextBuffer.module.exports.TextBuffer.save (/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:1108:25)
    at TextEditor.module.exports.TextEditor.save (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:925:32)
    at Pane.module.exports.Pane.saveItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:780:24)
    at Pane.saveItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:3:65)
    at Pane.module.exports.Pane.saveActiveItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:763:25)
    at Workspace.saveActivePaneItem (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:1548:46)
    at atom-workspace.core:save (/Applications/Atom.app/Contents/Resources/app/src/register-default-commands.js:236:38)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:35)
    at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:65
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -3:58.1.0 core:paste (input.hidden-input)
     -3:54.8.0 core:copy (input.hidden-input)
     -3:54.2.0 core:save (input.hidden-input)
  2x -3:48.3.0 core:close (input.hidden-input)
  2x -1:30.5.0 core:backspace (input.hidden-input)
     -1:29.9.0 core:move-down (input.hidden-input)
  2x -1:29.5.0 core:move-right (input.hidden-input)
  2x -1:29 core:backspace (input.hidden-input)
     -1:28.2.0 core:move-down (input.hidden-input)
  2x -1:28.1.0 core:move-right (input.hidden-input)
  2x -1:27.5.0 core:backspace (input.hidden-input)
     -1:26.9.0 core:save (input.hidden-input)
     -1:26.2.0 core:close (input.hidden-input)
     -0:42.5.0 core:backspace (input.hidden-input)
     -0:42.2.0 editor:newline (input.hidden-input)
  2x -0:36.8.0 core:save (input.hidden-input)
```

### Non-Core Packages

```
atom-bootstrap4 1.4.0 
atom-html-preview 0.1.22 
php-twig 4.0.0 
Remote-FTP 1.0.0 
```

**Comment by icetee** [comment]
Thanks, but duplicate and resolved #840 
Please wait for the new version!

**Comment by djit** [comment]
Thanks!
